### PR TITLE
Show Floating IP Suffix / Prefix Name

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:
Show the resulting complete Floating Pool name if user needs to specify a prefix or suffix for a Floating Pool.

Screen Shot 2022-10-19 at 15 42 06
Screen Shot 2022-10-19 at 15 42 15

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```feature user
Open Stack: Improved Floating IP wildcard configuration
```

### CODE REVIEW

File: frontend/src/components/WildcardSelect.vue [hunk 1]
```diff
@@ -133,6 +133,9 @@ export default {
       if (this.wildcardSelectedValue.customWildcard) {
         return `Specify custom ${this.wildcardSelectLabel}`
       } else {
+        if (this.wildcardVariablePart?.length) {
```
Comment (holgerkoser) [inline frontend/src/components/WildcardSelect.vue:136]:
```suggestion
        if (this.wildcardVariablePart) {
```

File: frontend/src/components/WildcardSelect.vue [hunk 2]
```diff
@@ -86,76 +97,62 @@ export default {
...
-      if (!this.wildcardSelectedValue.isWildcard) {
-        return undefined
-      }
-      if (this.wildcardSelectedValue.customWildcard) {
-        return `Specify custom ${this.wildcardSelectLabel}`
-      } else {
-        const label = this.wildcardSelectedValue.startsWithWildcard ? 'prefix' : 'suffix'
-        return `Selected wildcard value requires a ${label} which needs to be specified`
-      }
+      return `Floating Pool Name: ${this.internalValue}`
```
Comment (petersutter) [inline frontend/src/components/WildcardSelect.vue:120]:
```suggestion
      return `Custom Name: ${this.internalValue}`
```
maybe?

Comment (grolu) [reply frontend/src/components/WildcardSelect.vue:120]: yes. I just noticed hard coding it to Floating IP is a bad idea anyway as this component is kind of generic even if we currently only use it for floating IP. I'll double check this.

**Review by petersutter — APPROVED** [review]
/lgtm

**Review by holgerkoser — APPROVED** [review]
/lgtm

**Pull request merged**